source/codegen/metadata/nidaqmx/enums.py

```diff
@@ -1,2 +1,60 @@
 # -*- coding: utf-8 -*-
-enums = {}
+enums = {
+    'InputTermCfgWithDefault': {
+        'name': 'InputTermCfgWithDefault',
+        'values': [
+            {
+                'name': 'Cfg_Default',
+                'value': -1
+            },
+            {
+                'name': 'RSE',
+                'value': 10083
+            },
+            {
+                'name': 'NRSE',
+                'value': 10078
+            },
+            {
+                'name': 'Diff',
+                'value': 10106
+            },
+            {
+                'name': 'PseudoDiff',
+                'value': 12529
+            }
+        ]
+    },
+    'LineGrouping': {
+        'name': 'LineGrouping',
+        'values': [
+            {
+                'name': 'ChanForAllLines',
+                'value': 1
+            },
+            {
+                'name': 'ChanPerLine',
+                'value': 0
+            }
+        ]
+    },
+    'VoltageUnits2': {
+        'name': 'VoltageUnits2',
+        'values': [
+            {
+                'documentation': {
```

Comment:
Note: The enums with documentation I was able to generate from CVI enum metadata. The ones *without* documentation showed up in the *function* metadata (as a list of valid values for the parameter, essentially) but weren't separately documented as an enum.